.claude/commands/settings-integration.md

```
@@ -0,0 +1,597 @@
+if (mountPoint) {
+    createRoot(mountPoint).render(<MyApp />);
+}
+```
+
+---
+
+## Architecture Overview
+
+The integration has two layers that must stay in sync:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)